js/localstorage.js

```diff
@@ -0,0 +1,23 @@
+function localData() {
+  const localData = {
+    name: document.getElementById('savename').value,
+    email: document.getElementById('saveemail').value,
```

Comment:
- [ ] 👋 Thank you for your efforts in addressing the previous review comments. However, it seems that a new error has arisen in your code. Previously, you had an email input field with the ID "saveemail," which you updated to "email" after the review feedback. Now, when attempting to add data, an error occurs: "Cannot read properties of null (reading 'value') at HTMLInputElement.localData (localstorage.js:4:48)."

[Portfolio-Complete (1).webm](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/55840999/b402b9b4-e871-42f3-a3a0-af81271fa5ac)


Please resolve this issue 🙏, as it is preventing me from checking the updates made by the previous reviewer. By resolving this error, you'll be able to proceed with the further review of your changes. Keep up the great work, and happy coding! 💻🌟🚀